### ISSUE TITLE
migration documentation does not mention about changes related to range fields

### ISSUE THREAD

**Issue by KushGoyal** [carltongibson/django-filter#951]
I recently migrated to 2.0 and followed the migration guide. But it had not mentioned anything about _0 being changed to _after and _1 to _before for date and datetime range fields.
Similarly changes for number range field from _0 to _min and _1 to _max.

**Comment by rpkilby** [comment]
Good point. This is discussed [here](http://django-filter.readthedocs.io/en/master/guide/migration.html#multiwidget-replaced-by-suffixedmultiwidget-770), but the changes aren't called specifically in their own section.

**Comment by KushGoyal** [comment]
I think this needs a specific section since it is a breaking change. In fact the [docs](http://django-filter.readthedocs.io/en/master/ref/filters.html#datefromtorangefilter) are not very clear about it either. This is explained via examples. But for clarity I think it should be mentioned that lower bound is `<field_name>_after` and upper bound is `<field_name>_before`
